src/main/scala/firrtl/annotations/JsonProtocol.scala

```diff
@@ -217,7 +350,8 @@ object JsonProtocol {
       new LoadMemoryFileTypeSerializer + new IsModuleSerializer + new IsMemberSerializer +
       new CompleteTargetSerializer + new TypeSerializer + new ExpressionSerializer +
       new StatementSerializer + new PortSerializer + new DefModuleSerializer +
-      new CircuitSerializer + new InfoSerializer + new GroundTypeSerializer
+      new CircuitSerializer + new InfoSerializer + new GroundTypeSerializer +
+      new UnrecognizedAnnotationSerializerSerializer
```

Comment:
Note that this matches on every possibly annotation. I suspect that whatever is last will always be the lowest priority, but are we absolutely certain of that fact? I think there should at minimum be a comment here to prevent someone from adding a Serializer after it which would never be used because this one will always match.